openassessment/workflow/serializers.py

```diff
@@ -19,7 +20,8 @@ class Meta:
             'modified',
 
             # Computed
-            'score'
+            'score',
```

Comment:
Assuming we don't care about order, we can minimize the diff by adding the new value on a new line above this one:
```python
fields = (
    # ...
    override_score,
    score
)
```